docs/migrations/rest.md

```diff
@@ -0,0 +1,89 @@
+# REST Endpoints Migration
+
+Migrate your REST endpoints to the Stargate ones. {synopsis}
```

Comment:
Why do we have `{synopsis}`?

Reply:
It shows a nice little box 😮  https://docs.cosmos.network/master/basics/app-anatomy.html